datastore/importers/nmdb-import-cisco-show-vrf/Parser.hpp

```diff
@@ -0,0 +1,103 @@
+// =============================================================================
+// Copyright 2024 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS). Under the terms of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+// =============================================================================
+// Maintained by Sandia National Laboratories <Netmeld@sandia.gov>
+// =============================================================================
+
+#ifndef PARSER_HPP
+#define PARSER_HPP
+
+#include <netmeld/datastore/parsers/ParserHelper.hpp>
+#include <netmeld/datastore/objects/Vrf.hpp>
+
+namespace nmdo = netmeld::datastore::objects;
+namespace nmdp = netmeld::datastore::parsers;
+
+// =============================================================================
+// Data containers
+// =============================================================================
+
+struct Data
+{
+  std::vector<nmdo::Vrf> vrfs;
+
+  auto operator<=>(const Data&) const = default;
+  bool operator==(const Data&) const = default;
+};
+
+typedef std::vector<Data>    Result;
```

Comment:
If this is ONLY storing/processing a singular data type, then there is no need for all this. It should more simply be:
```suggestion
typedef std::vector<nmdo::Vrf>    Result;
```

You of course then have to fix the grammar, but it should be fairly straightforward.  If you need an example, you can see `nmdb-import-hosts`.  There are a couple other tools that do this as well if you need more examples.